csharp/ql/test/library-tests/dataflow/signanalysis/MissingSign.expected

```diff
@@ -0,0 +1,3 @@
+| SignAnalysis.cs:448:22:448:29 | Byte* to = ... |
```

Comment:
@hvitved Do you have any good idea how to handle these cases? The pointer has no sign, because the `= ...` is not a numeric or char type, but pointer is now considered as such.

Reply:
The problem is that the RHS is of type `byte[]` (which does not have a sign), but the whole assignment expression has pointer type. Arguably, the extractor should have added an implicit `(byte*)` cast on the RHS, in which case things should work.

Reply:
Yes, let's do it in a separate PR.